data/projects/bear_updates.md

```diff
@@ -0,0 +1,36 @@
+This project improves some of our bears.
```

Comment:
I just made a description for this on the wiki, wasn't aware you already had that, can you merge in my contents? I think the milestones are at least better defined